Integrations/EWSv2/EWSv2.py

```diff
@@ -873,7 +878,7 @@ def get_last_run():
         last_run = {
             LAST_RUN_TIME: None,
             LAST_RUN_FOLDER: FOLDER_NAME,
-            LAST_RUN_IDS: None
+            LAST_RUN_IDS: []
```

Comment:
Backwards compatibility: This will still be `None` on customer envs, make sure you handle both cases.